dcos/dcos-ui
# DCOS-18090: Remove Tabs from UI Components

Previously, each UI component in the design system had 3 tabs: overview, code and style. This content is now consolidated into a single page as requested by the design team.

To test:
- Enable the design system in Config.dev.js
```
        tracking: {
          enabled: true
        },
        "design-system": {
          enabled: designSystemEnabled
        }
```
- `useUIConfigFixtures` in `Config.dev.js` should be set to `true`
- Navigate to each of the components in the Components heading for the Design System category on the sidebar
- Each is a single page (only Buttons is populated with actual content at the moment)

Closes DCOS-18090







**Checklist**
